styles.css

```diff
@@ -0,0 +1,98 @@
+/* add your CSS here */
+
+.outer-container {
+	display: flex;
+	justify-content: space-between;
+	align-content: space-around;
+	flex-wrap: wrap;
+}
+
+.card {
+	width: 300px;
```

Comment:
maybe look at `rem` or `%` to add some responsiveness or relative spacing based on screen size